cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -55,6 +50,24 @@ $receipt = @"
 "@
 $receipt_home = "${env:LOCALAPPDATA}\{{ app_name }}"
 
+if ($env:{{ disable_update_env_var }}) {
+  $install_updater = $false
+} else {
+  $install_updater = $true
+}
+
+$NoModifyPath = $false
+if ($env:{{ no_modify_path_env_var }}) {
+    $NoModifyPath = $true
+}
+
+$unmanaged_install = $env:{{ unmanaged_dir_env_var }}
+
+if ($unmanaged_install) {
+  $NoModifyPath = $true
+  $install_updater = $false
+}
```

Comment:
I think the "mode" variable should get applied first, and then the other overrides should kick in..?